AirCasting/Settings/BackendSettingsView.swift

```diff
@@ -53,6 +54,7 @@ struct BackendSettingsView: View {
         Button {
             urlProvider.baseAppURL = url ?? URL(string: "http://aircasting.org/api")!
             presentationMode.wrappedValue.dismiss()
+            try! logoutController.logout()
```

Comment:
We really should not crash on error here, let's handle it somehow

Reply:
For sure, nonsense, trying now